### ISSUE TITLE
[FEATURE] Ray HPO Tune Integration

### ISSUE THREAD

**Issue by yinweisu** [awslabs/aws-orbit-workbench#353]
We've managed to run an axample ray cluster and HPO on kubenetes. Here are the steps:
1. Connect local kubectl to the eks cluster
2. Pip install ray locally: ```pip install ray```
3. Clone ray repo: ```git clone https://github.com/ray-project/ray.git```
3. Launch up a ray cluster by: ```ray up ray/python/ray/autoscaler/kubernetes/example-full.yaml```
4. Check the cluster got launched by: ```kubectl -n ray get pods```
5. To create a sample job: ```kubectl create -f ray/doc/kubernetes/job-example.yaml```
     * You can modify the script got downloaded in the yaml file to run different script
6. To check result: ```kubectl -n ray logs <launched job pod>```
7.  To tear down the cluster: ```ray down ray/python/ray/autoscaler/kubernetes/example-full.yaml```

**Comment by abaror** [comment]
nice, you can turn this into a plugin,  make sure https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/kubernetes/example-full.yaml is changed to use the <team namespace> and not introduce a new namespace "ray" .  This will allow ray to launch containers within the team namespace.

**Comment by yinweisu** [comment]
I've created a sample notebook and the sample yaml file:
https://github.com/yinweisu/RayOrbit

You'll have to launch up the ray cluster in the terminal instead of in the notebooks because it is an interactive command, and I'm not sure how do you interact using a notebook shell :(.

**Comment by sxjscience** [comment]
@yinweisu Would commands like the following work?

```
ray up -y ...
```

**Comment by yinweisu** [comment]
@sxjscience Yes. You are correct. I've updated the notebook. Thanks for the help!

**Comment by sxjscience** [comment]
@abaror Weisu has updated the notebook and you may try that out. There is also an explanation about what ray/tune integration can bring to Orbit.

**Comment by abaror** [comment]
we are adding support for helm charts (https://helm.sh/docs/topics/charts/) as plugins.  As soon as we have it ready , we can add this as a new plugin using the new framework. About a week time.
